tp26/README.md

```diff
@@ -0,0 +1,140 @@
+The `delegation` condition contains enough complexity that it deserves a separate technical proposal. There are mentions of delegates and delegation in this TP, but its nuance is not fully captured. For the curious, delegation allows the following scenario: if Alice gives Bob a grant with `delegation` enabled, then Bob can issue grants of the same or narrower scope to Carol.
+
+### Type Definitions
+See the type definitions below for a list of `scope` and `conditions` options. More options may be added in the future.
+```typescript
+type PermissionsScope = {
+  interface: DwnInterface;
+  method: DwnMethod;
+  // The grantee may access records with matching `schema`
+  schema?: string;
+  // The grantee may access protocols or records with matching protocol URI.
+  protocol?: string;
+  // The grantee may access a specific records with matching recordId.
+  recordIds?: string[];
+};
+
+type PermissionsConditions = {
+  // Determines whether the permissioned message must, may, or must not be signed.
+  attestation?: 'required' | 'optional' | 'prohibited';
+  // Determines whether a record must, may, or must not be signed.
+  encryption?: 'required' | 'optional' | 'prohibited';
+  // If true, the grantee may mark a record as published. If false or omitted, they may not.
+  publication?: boolean;
+  // If true, the grantee may write a subsequent `PermissionsGrant`, giving the same or narrower access
+  // to another entity. If false or omitted, the grantee may not write issue their own `PermissionGrant`.
+  // We use an enum instead of a boolean because we may add more options for delegation in the future.
+  delegation?: 'prohibited' | 'allowed';
+  // If true, the grantee may access any objects or data within the granted capabilities, including objects or
+  // data that it has not created. If false or omitted, the the grantee may only access objects or data that it has created.
+  sharedAccess?: boolean;
+};
+```
+
+
+## Messages
+### PermissionsRequest
+By themselves `PermissionsRequests` do nothing. However, it is necessary for the DWN spec and its reference implementation to define them. `PermissionsRequest` serves as an "inbox" for requests.
+2. We must standardize how one entity to requests access to another entity's resources. If we do not define this, we risk each app defining their own access-request pattern with no interoperability between apps.
+
+### PermissionsGrant
+`PermissionsGrant` explicitly gives an external entity the ability to send a DWN message within some `scope`; further restrictions are defined in the `conditions` property. To use a `PermissionsGrant`, the external entity must include the CID of the relevant `PermissionsGrant` in the `permissionsGrantId` field of each message's `authorization`. The grant is active until its `expiresAt` time or until a `PermissionsRevoke` is affected.
+
+`PermissionsGrant` messages will have an `encryptionKey` property if the `encryption` condition is `required` or `optional`.
```

Comment:
Would love more elaboration. Fine if this belongs to a separate TP.

Reply:
I was going off the DWN spec for this point. Here's the paragraph I was drawing from. Does this answer your questions?

> The message MUST contain an encryptionKey property if the data transacted using the permission grant is to be encrypted, per the directives for encryption under the encryption field of the permission’s [conditions](https://identity.foundation/decentralized-web-node/spec/#permission-conditions). If present, the value of the encryptionKey property MUST be a [[RFC7516](https://identity.foundation/decentralized-web-node/spec/#ref:RFC7516)] JSON Web Encryption (JWE) object that contains the encrypted key material required for an authorized party to decrypt the JWE represented by the dataCid value within the descriptor object.

Reply:
Ah I see what you mean. I'll reword that sentence and update `encryptionKey` -> `encryption`.